doc/index.rst

```diff
@@ -1,10 +1,44 @@
-The Plutus Platform
+Plutus Tools SDK User Guide
```

Comment:
Hmm, do we need to duplicate the title here? Can we get sphinx to print it here or something?

Reply:
Good question. When I removed the H1 heading here, it also impacted the title displaying above. I would think there would be a way to avoid the duplication. I'll look into that.

Reply:
In the current versions of both RTD instances, the title is duplicated. I would like to find a way to not duplicate it, but just want to note that this isn't a new issue. But I'll look into this to see if we can improve this behavior.